src/startupscriptgenerator/src/common/scriptBuilder.go

```diff
@@ -0,0 +1,50 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+	"strings"
+)
+
+type ScriptBuilder struct {
+	scriptBuilder strings.Builder
```

Comment:
member `scriptBuilder` in type `ScriptBuilder` seems redundant - maybe just `script`?